Rakefile

```diff
@@ -0,0 +1,181 @@
+require 'rake'
+
+require ::File.expand_path('../config/environment', __FILE__)
+
+# Include all of ActiveSupport's core class extensions, e.g., String#camelize
+require 'active_support/core_ext'
```

Comment:
You should try and require the most specific thing that you need. As apposed to requiring the kitchen sink.